src/services/appointmentData.service.js

```diff
@@ -2,11 +2,22 @@ import { isAvailable } from "../components/FilterPanel/AvailabilityFilter";
 import { isWithinRadius } from "../components/FilterPanel/RadiusFilter";
 
 const dayjs = require("dayjs");
+const customParseFormat = require("dayjs/plugin/customParseFormat");
```

Comment:
why are we using a mix of requires and imports?

Reply:
No reason. I just copy/pasted from dayjs documentation.  It's update in the latest commit!